### PR TITLE
Register raikan-san.is-a.dev

### PR DESCRIPTION
**Pull request by wrothmir**
Register raikan-san.is-a.dev with CNAME record pointing to raikan-san.github.io.

### CODE REVIEW

File: domains/raikan-san.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "raikan-san",
+           "email": "",
+           "discord": "459219626484826113",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.ngpy9YsUgKD0qcHpKxy8pPkaTg_faaoWhEIYGyzU2hlFPcOR26gOFA028IbhZ3BlwJNqz6hDjTMdbI4nWkq9DtAelJmmyDGNgLkXhdEZFiTjx0cL4O_xX57QZlBkO8NiklF3YfP0MarhqVlaToRcZjq5kMrX5P0LaDBZadBmjRv7nKexueYa4r5-OxjSN2IPCVmiTURL8Ccaqbzv1bscqKXFh8g36PzGobs1PBv3GUcMIjy4BzbpGWAsOS6XHNPw2ts1mAjodvmAlUjlxpSfuzcLsBoJH4FEg5Q1l9h_vz5enR0prM1puPasuaB_Na4tedtV7UIr98TjZty048ctMA.aZEcqmCNdgV2kSYAWJ7UaA.EQcnhLWSHb0wQeSA_0EtWdBun6C8sE-gBMcwfAmcxReM0TyanIl4dO-KqF4v2jqzHP--oRJm3FAX0ALjWiLWj83j48KMwlKMBIBBTCYTRgmOpDsnXaZbbzoiAJvIYGGU.LTgA1lo8oNWZt6HFDzhm0Q"
+        },
+    
+        "record": {
+            "CNAME": "raikan-san.github.io"
```
Comment (DEV-DIBSTER) [inline domains/raikan-san.json:10]: Little content displayed on the site.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/raikan-san.json (hunk 1 above)
Comment (raikan-san) [reply domains/raikan-san.json:10]: I'm working on the website right now, will take a bit of time to get stuff up on it.
Should I let you know when there is content on it @DEV-DIBSTER ?

Comment (DEV-DIBSTER) [reply domains/raikan-san.json:10]: Yeah, this is what I'd recommend. Closing this PR, and when you finish making changes, you can reopen the PR, and you can @ me, and I'll review it. 🫡

Comment (raikan-san) [reply domains/raikan-san.json:10]: 🫡

Comment (raikan-san) [reply domains/raikan-san.json:10]: @DEV-DIBSTER I've made changes to the website. I am not sure how to go about reopening this PR. I don't see an option for it anywhere. Let me know what I can do to reopen it whenever you can! 🖖🏽

Comment (CuteDog5695) [reply domains/raikan-san.json:10]: You know you can make another PR, right?

Comment (raikan-san) [reply domains/raikan-san.json:10]: @DEV-DIBSTER I just opened a new PR because I couldn't reopen this one. Hope that's okay 🫡

**Pull request closed without merging**